easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb

```diff
@@ -2939,11 +2939,11 @@ exts_list = [
     ('cSEM', '0.4.0', {
         'checksums': ['7753ac7db9d2c0392e51dd31ec8638e1a7fcbb2546dd9103f5ecc03dd51836c1'],
     }),
-    ('cubelyr', '1.0.1', {
```

Comment:
Please restore this extension.

Reply:
Muh, no idea how that happened... Will fix.